src/main/java/org/synyx/urlaubsverwaltung/calendar/DepartmentCalendar.java

```diff
@@ -24,6 +27,7 @@
 
     @NotNull
     @Column(name = "department_id")
+    @OnDelete(action = CASCADE)
```

Comment:
This is like a marker annotation to inform the developer what is created via liquibase and would be created via hibernate auto creation e.g.